src/components/shared/input/Input.tsx

```diff
@@ -1,14 +1,21 @@
 import "./Input.scss";
+import TextField from "@mui/material/TextField";
+import InputLabel from "@mui/material/InputLabel";
 
 export interface InputProps {
   label: string;
 }
 
 export const Input = ({ label }: InputProps) => {
   return (
-    <label className="label" htmlFor={label}>
-      {label}
-      <input id={label} placeholder={label} />
-    </label>
+    <>
+      <InputLabel>{label}</InputLabel>
+      <TextField variant="filled" />
```

Comment:
What if I want to reuse it with different variant?

Reply:
removed variant.